src/components/film-details-information/film-details-information.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { formatRunTime } from './helper';
+
+export const FilmDetailsInformation = ({
+  director, starring, runTime, genre, released,
+}) => (
+    <>
+      <div className="movie-card__text movie-card__row">
+        <div className="movie-card__text-col">
+          <p className="movie-card__details-item">
+            <strong className="movie-card__details-name">Director</strong>
+              <span className="movie-card__details-value">{director}</span>
+          </p>
+          <p className="movie-card__details-item">
+            <strong className="movie-card__details-name">Starring</strong>
+            <span className="movie-card__details-value">
+              {starring.map((item, index) => {
+                if (index === starring.length - 1) {
+                  return <>
+                    {item}
+                  </>;
+                }
+                return <>
```

Comment:
so difficult to read.

Please use this 


```suggestion
                return (
                  <Fragment key={id}>
                    {item}, <br></br>
                  </Fragment>
                );
```